feathr_project/test/test_azure_spark_e2e.py

```diff
@@ -106,7 +104,7 @@ def test_feathr_online_store_non_agg_features():
         __file__).parent.resolve() / "test_user_workspace"
     client = basic_test_setup(os.path.join(test_workspace_dir, "feathr_config.yaml"))
 
-    online_test_table = get_online_test_table_name('nycTaxiCITable')
+    online_test_table = get_online_test_table_name('nycTaxiCITableNonAggFeature')
```

Comment:
It would be better if we have a centralized place to maintain all the test table names.